src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -436,34 +439,51 @@ QueryNetworkResult Plugin::QueryNetwork(const CNNNetwork& network,
             return false;
         }
         if (ngraph::is_type<const ngraph::op::v0::Constant>(node)) {
-            constantsNames.emplace(node->get_friendly_name());
             constants.push_back(node);
             return false;
         }
-        return prog.IsOpSupported(network, node) &&
-               !ngraph::op::is_parameter(node) &&
-               !ngraph::op::is_output(node);
+        return prog.IsOpSupported(network, node) ||
```

Comment:
Why did you change the logic?

Reply:
Some operations may be fused to parameter or output. 
In this code all parameter and outputs are dropped and some fused operations may be lost here.

There is some additional logic to save all fused names for constants. I kept it. But for parameters and outputs I change logic and mark them as supported w/o any additional checks.

Reply:
@vladimir-paramuzov Please take a look

Reply:
Additional conformance validation doesn't show any degradation.